xshop/users/views.py

```diff
@@ -1,27 +1,24 @@
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework import status
-from .serializers import UserSerializer , LoginSerializer
-from django.contrib.auth.models import User
-
+from .serializers import UserSerializer, LoginSerializer
 from rest_framework import generics
 
-from . import models
-
 
 class UserCreate(APIView):
-
-    def post(self, request, format='json'):
+    def post(self, request, format="json"):
         serializer = UserSerializer(data=request.data)
         if serializer.is_valid():
             user = serializer.save()
             if user:
                 return Response(serializer.data, status=status.HTTP_201_CREATED)
 
+
 class LoginAPIView(generics.GenericAPIView):
     serializer_class = LoginSerializer
+
     def post(self, request):
         serializer = self.serializer_class(data=request.data)
         serializer.is_valied(raise_exception=True)
```

Comment:
Here you get the token of the user from the db and return in it in the response as described below